proto/felixbackend.proto

```diff
@@ -272,6 +272,13 @@ message ServiceAccountMatch {
 
 message HTTPMatch {
   repeated string methods = 1;
+  message PathMatch {
+    oneof path_match {
+      string exact = 2;
+      string prefix = 3;
+    }
+  }
+  repeated PathMatch paths = 4;
```

Comment:
Just noticed this numbering is off.  The numbers are scoped to the message, so exact should be 1, prefix 2, and paths 2.